test/tasks/setup.rake

```diff
@@ -5,7 +5,7 @@ namespace :pm do
       Dir.chdir('./test') do
         db_type = args[:db_type] || 'postgresql'
         `rm -rf testapp`
-        `bundle exec rails new testapp -f -d #{db_type} -m ./template.rb  --skip-keeps --skip-spring  --skip-git`
+        `bundle exec rails new testapp -f -d #{db_type} -m ./template.rb  --skip-keeps --skip-spring  --skip-git --skip-javascript`
```

Comment:
why are we skipping javascript here?

Reply:
Rails 6 by default uses Webpacker which was causing some errors so we added that to disable it.